### DISCUSSION TITLE
AVR16EBxx seems to be shipping

### DISCUSSION THREAD

**Discussion by leo60228** [SpenceKonde/DxCore#503] [General]
Microchip Direct seems to have quite a few 16KB EB parts in stock, and a [Preliminary Data Sheet](https://ww1.microchip.com/downloads/aemDocuments/documents/MCU08/ProductDocuments/DataSheets/AVR16EB14_20_28_32-Prelim-DataSheet-DS40002522.pdf) is also available, along with an [AVR16EB32 Curiosity Nano](https://www.microchipdirect.com/dev-tools/EV73J36A). The datasheet also has mentions of 32KB parts, but the 8KB parts seem to have vanished.

**Comment by SpenceKonde** [comment]
Yeah 8k ones got axed. Microchips promised AVR16EB freebies hacen;t arrived for me. and you know I can't upload to EA's currently right? Yeah so all the EA stuff is tested entirely by other people and seems to work (not surprising since it was nothing novel). That's not the case with the EB - wrangling those timers means I have to actually be able to upload code to the device, meaning modification to serialUPDI. Someone offered me the information on how to do it but I have been so busy I never got back to him

**Comment by leo60228** [comment]
Did you see that there was a pymcuprog update a few weeks ago that allegedly fixes serialupdi on Ex?

**Comment by SpenceKonde** [comment]
No I was busy working on gathering data for my tax preparer after getting a nastygram from the IRS.
